MenubarlessClock/MBLCAppDelegate.m

```diff
@@ -373,6 +384,7 @@ - (void)	adaptUIToDarkMode
 		self.window.backgroundColor = [NSColor colorWithWhite: 1.0 alpha: 0.9];
 		self.timeField.textColor = [NSColor blackColor];
 	}
+    [self updateClock:nil];
```

Comment:
Here I intended to trigger battery icon invalidation but couldn't find a better way. 
Please let me know if you can think of a better one!